.gitignore

```diff
@@ -9,3 +9,4 @@ _scratch/
 /spec/gen_spec/spec*.db
 tags
 /tests/debug.py
+.idea/
```

Comment:
An ignore line like this belongs in your user level .gitignore or above, not at the project level. Only ignore items specific to this project belong here. My user gitignore is at `~/.config/git/ignore`, your's may be in a different location depending on your OS etc.